django/contrib/admin/static/admin/js/vendor/select2/select2.full.js

```diff
@@ -1098,7 +1098,7 @@ S2.define('select2/results',[
 
       var currentIndex = $options.index($highlighted);
 
-      // If we are already at te top, don't move further
+      // If we are already at the top, don't move further
```

Comment:
We shouldn't make changes to vendored code. I submitted this upstream in https://github.com/select2/select2/pull/5336.